src/containers/organizations/components/OrganizationsComponent.js

```diff
@@ -213,57 +178,83 @@ class OrganizationList extends React.Component {
       if (!newSelectedOrgOption) {
         hashHistory.push('/org');
       }
-      else if (this.props.selectedOrgOption.value !== newSelectedOrgOption.value) {
+      else if (!this.props.selectedOrgOption || this.props.selectedOrgOption.value !== newSelectedOrgOption.value) {
         hashHistory.push(`/org/${newSelectedOrgOption.value}`);
       }
     };
 
     return (
       <Select
+          className={classnames(styles.organizationsSelect)}
           clearable
           searchable
-          placeholder="Your Organizations"
           options={this.props.selectOrgOptions}
           value={this.props.selectedOrgOption}
           onChange={updateSelectedOrganization} />
     );
   };
 
-  render() {
+  renderCreateOrganizationComponent = () => {
+
+    return (
+      <CreateOrganization
```

Comment:
One side note: I've started making the `Create...` components smart components. It makes it a lot easier to move them around in the UI when redesigns happen, and keeping the async creation logic isolated to them is convenient.